src/braket/default_simulator/openqasm/interpreter.py

```diff
@@ -447,6 +447,7 @@ def _(self, node: QuantumGate) -> None:
 
     @visit.register
     def _(self, node: QuantumPhase) -> None:
+        self._uses_advanced_language_features = True
```

Comment:
Hm, this was previously not counted here because the gates were defined in terms oq3 built-in gates so there would be false positives. Might be safe to have this now that the gates get simulated directly.